### ISSUE TITLE
highlight next speaker dot on debate page

### ISSUE THREAD

**Issue by jakubmanczak** [debatecore/debate-tools#29]
based on musketeers of words marshall feedback:

- the dots flashing green are nice
- it might be nice if the dot was solid green when the speaker is the next one up

**Comment by Mateusz-Dobrzynski** [comment]
It's a neat idea. Do we want it to change during the preceding speech or after it but before the next speech is started?

**Comment by jakubmanczak** [comment]
While the clock isn't running - having two dots be highlighted at once would be confusing, I think.

**Comment by jakubmanczak** [comment]
I don't think this works as well as it sounds.
Maybe a different color would be better? White, mayhaps?

https://github.com/debatecore/debate-tools/assets/56172798/3fc70d64-2754-4ff2-bef8-e2124680d550

**Comment by Mateusz-Dobrzynski** [comment]
You're right, a static green dot gives the impression that the animation is broken. A dot of a different color would be unambiguous in this regard.
